compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -238,12 +237,11 @@ pub fn target_features(sess: &Session) -> Vec<Symbol> {
             .filter_map(|&(feature, gate)| {
                 if sess.is_nightly_build() || gate.is_none() { Some(feature) } else { None }
             })
+            .flat_map(|feature| to_llvm_features(sess, feature))
```

Comment:
This change alters the behaviour of the function entirely. Previously it was returning the _rust_ view of the target features (some of which use different names, and some of which have mapping that's not 1:1), and now it'll return a list of the backend feature names.

I believe (though a grain of salt: I'm going entirely from my memory of this code) this makes this change incorrect (e.g. we'd end up setting `cfg(target_feature="rdrnd")` instead of `cfg(target_feature="rdrand")`)

Reply:
Mmm, I thought that was what this function was for, actually?